modules/rhizo_base/templates/config_values.py.erb

```diff
@@ -31,7 +31,7 @@ vpn_ip_address = "<%= @vpn_ip_address %>"
 wan_ip_address = "<%= @wan_ip_address %>"
 riak_ip_address = "<%= @riak_ip_address %>"
 mncc_ip_address = "172.16.0.1"
-sip_central_ip_address = <%= @sip_central_ip_address %>
+sip_central_ip_address = "<%= @sip_central_ip_address %>"
```

Comment:
Actually that will break when the value is a list (which it is with TIC)

Reply:
Ah, that makes sense. I figured this might be the case. I will change the default hiera parameter then to be a one item list instead of a value.

Reply:
It's also fine as a value. The code will check before using it.